src/main/scala/mesosphere/marathon/MarathonConf.scala

```diff
@@ -224,4 +225,13 @@ trait MarathonConf
     descr = "The timeout for preparing the Marathon instance when elected as leader.",
     default = Some(3 * 60 * 1000L) //3 minutes
   )
+
+  lazy val maxQueuedRootGroupUpdates = opt[Int]("max_queued_root_group_updates",
```

Comment:
Can we name all our internal parameters `internal` - so it is easy to reason about (the val and the cmd line parameter)?